### ISSUE TITLE
Non-issue, but system specifications for PL in MPSoC

### ISSUE THREAD

**Issue by BeethovenKodar** [forsyde/IDeSyDe#31]
Platform:
[MPSoC_Intermediate.txt](https://github.com/forsyde/IDeSyDe/files/15075108/MPSoC_Intermediate.txt)

Application:
[ToySDF_Intermediate.txt](https://github.com/forsyde/IDeSyDe/files/15075137/ToySDF_Intermediate.txt)

Rename the file extensions to `.fiodl` and you should be all set!

**Comment by Rojods** [comment]
Thanks! Will try out the files soon.

**Comment by Rojods** [comment]
Is it intentional that the Cores in the MPSoC have TDMAs schedulers? It could have been my bad explanation in some of our physical meetings! 

If you wanted the cores to be "bare-metal" in the sense that @ingo-sander usually means, the correct trait is `SuperLoopRuntime`. 

I am asking because _it is perfectly possible to account for TDMA, but there won't be any schedulers in the final result, only time-slots which will determine how the CPU is switching between processes.

For the moment, I will assume for simplicity that you meant `SuperLoopRuntimes` and act accordingly.

**Comment by BeethovenKodar** [comment]
Yeah I just picked some scheduler that I found in some demo file I believe. This was a long time ago, but sure go for the `SuperLoopRuntimes`!

**Comment by BeethovenKodar** [comment]
@Rojods Made some slight changes to the files. They use `SuperLoopRuntime` by default and one of the actors is hw mappable onto two possible FPGAs to enable more options in the exploration. This exact input is explored by the current IDeSyDe develop branch successfully. However, you said that the PL input specifications got a bid from the new MiniZinc explorer, but I don't see that and instead, Jenetics handles the exploration. I may have misunderstood something though.
[MPSoC_Intermediate.txt](https://github.com/forsyde/IDeSyDe/files/15210522/MPSoC_Intermediate.txt)
[ToySDF_Intermediate.txt](https://github.com/forsyde/IDeSyDe/files/15210523/ToySDF_Intermediate.txt)

**Comment by Rojods** [comment]
Now this has been solved through offline conversations and fixing the examples!
